src/content/documentation/publish/third-party-library-usage.md

```diff
@@ -15,7 +15,11 @@ date: 2022-10-17
 
 # Third Party Library Usage
 
-To complete the review process at [addons.mozilla.org](https://addons.mozilla.org) (AMO), reviewers must be able to verify the code in your extension. You may include third-party libraries in your extension. In that case, when you upload your extension to AMO, you will need to provide links to the library source code.
+> Only release versions of third-party libraries and/or frameworks may be included with an add-on. Modifications to these libraries/frameworks are not permitted.
```

Comment:
```suggestion
> Only release versions of third-party libraries and frameworks can be included in an add-on. Modifications to included libraries and frameworks are not permitted.
```

Reply:
Instead of duplicating the sentence, can we link to the policies section like:

`Please note the requirements for third-party libraries in the #Add-on Policies#`, where `#Add-on Policies`links to the development section of the policies.

Reply:
Sounds good. I'll update the PR to integrate this feedback after I merge @rebloor's other feedback.